### PR TITLE
 playbook 9,10

### PR DESCRIPTION
**Pull request by gaurav-calsoft**

### CODE REVIEW

File: ansible/playbooks/l3_single_routed_bgp_with_l2.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        pn_clipassword: "{{ PASSWORD }}"
+        pn_switch_list: "{{ groups['switch'] }}"
+        pn_trunk_data: "{{ lookup('file', '{{ trunk_file }}') }}"
+
+    - name: Create vrouter
+      pn_vrouter_creation:
+        pn_cliusername: "{{ USERNAME }}"
+        pn_clipassword: "{{ PASSWORD }}"
+        #pn_loopback_ip: '10.10.10.10'
+        #pn_vrrp_id: '18'
```
Comment (ashxjain) [inline ansible/playbooks/l3_single_routed_bgp_with_l2.yml]: We don't need vrrp here.

File: ansible/playbooks/l3_single_routed_ospf_with_l2.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        pn_clipassword: "{{ PASSWORD }}"
+        pn_switch_list: "{{ groups['switch'] }}"
+        pn_trunk_data: "{{ lookup('file', '{{ trunk_file }}') }}"
+
+    - name: Create vrouter
+      pn_vrouter_creation:
+        pn_cliusername: "{{ USERNAME }}"
+        pn_clipassword: "{{ PASSWORD }}"
+        pn_loopback_ip: '10.10.10.10'
+        pn_vrrp_id: '18'
```
Comment (ashxjain) [inline ansible/playbooks/l3_single_routed_ospf_with_l2.yml]: We don't need vrrp here.

File: ansible/playbooks/l3_single_routed_bgp_with_l2.yml (hunk 1 above)
Comment (ashxjain) [inline ansible/playbooks/l3_single_routed_bgp_with_l2.yml]: Also why are these things commented here and not in other playbook?

File: ansible/playbooks/l3_single_routed_ospf_with_l2.yml (hunk 2 above)
Comment (gaurav-calsoft) [inline ansible/playbooks/l3_single_routed_ospf_with_l2.yml]: done

**Pull request merged**